src/interfaces/IBridger.sol

```diff
@@ -37,20 +37,22 @@ interface IBridger {
     error SwapsDisabled();
     error NotEnoughEthToBridge();
     error GasFeeTooHigh();
-    error ApprovalFailed();
     error SwapCallFailed();
     error SlippageError();
+    error OnlyExchangeProxy();
 
     /* ============ Structs ============ */
 
     struct SignatureData {
+        address kintoWallet; // Kinto Wallet Address on L2 where tokens will be deposited
         address signer;
         address inputAsset;
-        uint256 amount;
         address finalAsset;
+        uint256 amount;
         uint256 nonce;
         uint256 expiresAt;
         bytes signature;
+        SwapData swapData; // Struct with all the required information to swap the tokens
```

Comment:
It can take a while since the user signs and the signature is sent. swapData cannot be here but I would include minAmount from swapDta in here